packages/unity-interface/portableExperiencesUtils.ts

```diff
@@ -203,3 +214,20 @@ export async function spawnPortableExperience(
 
   return { pid: id, parentCid: parentCid }
 }
+
+export async function setDisabledPortableExperiences(idsToDisable: string[]) {
+  idsToDisable.forEach(async (pexId) => {
+    if (currentPortableExperiences.has(pexId)) {
+      await killPortableExperienceScene(pexId)
+    }
+  })
+
+  const profile = getCurrentUserProfile(store.getState())
+  profile?.avatar.wearables.forEach((wearableId) => {
+    if (!idsToDisable.includes(wearableId)) {
+      // TODO: SPAWN PORTABLE EXPERIENCE WITH ID = wearableId (if it is an equipped smart wearable)
```

Comment:
Hey @menduz could you give me a hand with this `TODO`? I need to get the equipped wearables info in order to be able to call the `spawnPortableExperience` function ans so start again the non-ignored PEXs. Thanks!